api/v1beta1/bgppeer_webhook.go

```diff
@@ -176,10 +176,10 @@ func (bgpPeer *BGPPeer) validateBGPPeerConfig(existingBGPPeersList *BGPPeerList)
 	}
 
 	for _, BGPPeer := range existingBGPPeersList.Items {
-		if bgpPeer.Name != BGPPeer.Name && remoteASN == BGPPeer.Spec.ASN && address == BGPPeer.Spec.Address && myASN == BGPPeer.Spec.MyASN {
+		if bgpPeer.Name != BGPPeer.Name && address == BGPPeer.Spec.Address {
 			return field.Invalid(field.NewPath("spec").Child("Address"), address,
-				fmt.Sprintf("Duplicate BGPPeer %s ASN %d in the same BGP instance",
-					address, remoteASN))
+				fmt.Sprintf("Duplicate BGPPeer %s in the same BGP instance not supported in FRR mode",
```

Comment:
This works for native? If so, I think we should use the bgp mode conditional for this and not fail when running in native mode like we do for `validateBGPPeersMyASN`.

Reply:
+1

Reply:
mots of the checks are for FRR only I will add the checj